.goreleaser.yaml

```
@@ -0,0 +1,297 @@
+      - src: ./uninstall.sh
+        dst: /opt/cloud-update/uninstall.sh
+        file_info:
+          mode: 0755
+    
+    scripts:
+      postinstall: |
+        #!/bin/sh
+        # Create config directory
+        mkdir -p /etc/cloud-update
+        chmod 700 /etc/cloud-update
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)